app/views/layouts/_navbar.html.haml

```diff
@@ -0,0 +1,7 @@
+- if user_signed_in?
+  -# = link_to "Hello #{current_user.name}", root_path
+  = link_to "Edit", edit_user_registration_path
```

Comment:
Is editing the profile part of this feature?

Reply:
It is not part of this feature right now. We'll comment it out until we need it.